test-server/test-echo.c

```diff
@@ -33,6 +33,9 @@
 #include <syslog.h>
 #include <sys/time.h>
 #include <unistd.h>
+#else
+#include "../win32port/win32helpers/gettimeofday.h"
```

Comment:
Yes, #include "gettimeofday.h" is much better and works out nicely.

The include as such is necessary though, otherwise I get the following warning:

w:/libwebsockets/test-server/test-echo.c: In function 'main':
w:/libwebsockets/test-server/test-echo.c:417:4: warning: implicit declaration of function 'gettimeofday' [-Wimplicit-function-declaration]
    gettimeofday(&tv, NULL);